login funcionando melhor agora
    -antes login podia falhar e o usuario ainda assim entrava como algum outro usuario

criacao de topicos melhorada
    -agora podem ser criados n topicos, antes era apenas 1

Problemas:

    - (consertado) quando um usuario publica, no momento em que outro se inscreve naquele topico,
ocorre um erro em 'notification_callback'. O content é recebido, mas não pode ser lido pela funcao (pode ser impresso num print)

    - se o usuario ja era inscrito antes, ocorre outro erro no callback (timeout na linha 20 de cliente.py - 
'result = self.client.exposed_publish(self.user_id, topic, content)')
        - erro provavelmente em '_notify_subscribers', não sei se 'subscriptions' tem callback ou se é acessível